budget/templates/layout.html

```diff
@@ -34,10 +34,15 @@
     </script>
 </head>
 <body>
+<nav class="navbar navbar-toggleable-md navbar-inverse fixed-top bg-inverse">
+      <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
 
-  <nav class="navbar navbar-toggleable-md navbar fixed-top navbar-inverse bg-inverse">
-  <h1 class="col-2"><a class="navbar-brand" href="{{ url_for(".home") }}">#! money?</a></h1>
-            <ul class="navbar-nav col-5 offset-md-1">
+      <a class="navbar-brand" href="#">#! money?</a>
```

Comment:
I think we should preserve semantic HTML tags. Here we are loosing the information that it's a title.